langchain/chains/conversation/base.py

```diff
@@ -51,10 +57,9 @@ def validate_prompt_input_variables(cls, values: Dict) -> Dict:
             )
         prompt_variables = values["prompt"].input_variables
         expected_keys = memory_keys + [input_key]
-        if set(expected_keys) != set(prompt_variables):
+        if set(expected_keys).difference(set(prompt_variables)):
```

Comment:
```suggestion
        if not set(expected_keys).issubset(set(prompt_variables)):
```

Reply:
Modify it to check if all expected_keys are in prompt_variables, but still allow prompt_variables to contain additional keys